roles/installer/templates/deployments/web.yaml.j2

```diff
@@ -362,6 +382,15 @@ spec:
               - key: bundle-ca.crt
                 path: 'bundle-ca.crt'
 {% endif %}
+{% if web_host_network %}
+          hostNetwork: {{ web_host_network }}
+{% endif %}
+{% if web_dns_policy %}
+         dnsPolicy: {{ web_dns_policy }}
+{% endif %}
+{% if web_dns_policy == 'None' and web_dns_config is defined }}
```

Comment:
Similarly, there is an extra closing brace at the end of this line too which causes syntax errors when applied.